Source/WTF/wtf/text/StringImpl.h

```diff
@@ -48,6 +48,12 @@ typedef const struct __CFString * CFStringRef;
 @class NSString;
 #endif
 
+#if HAVE(36BIT_ADDRESS)
+#define STRING_IMPL_SHAPE_16BYTE_ALIGNMENT alignas(16)
```

Comment:
nit: this should be called `STRING_IMPL_ALIGNMENT` since it's not always 16.

Reply:
Updated